manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+# 1.11.29#!/usr/bin/env python
```

Comment:
Not sure what happened here, but I'd like this change to be reverted back to what it was.